src/store/ShareStore.js

```diff
@@ -47,10 +47,8 @@ export default class ShareStore {
             window.location.origin : 'https://charts.binary.com';
         this.shareLink = `${origin}#${encodeURIComponent(json)}`;
 
-
         this.loading = true;
-
-        fetch(`${this.bitlyUrl}?access_token=${this.accessToken}&longUrl=${encodeURIComponent(this.shareLink)}`)
+        fetch(`${this.bitlyUrl}?access_token=${this.accessToken}&longUrl=${this.shareLink}`)
```

Comment:
you change to use `encodeURIComponent` from twice to once. I assume this is some fix or smth...

Reply:
it's already `encode` above, by re-encoding, it get damaged